hash_checks/optic_checks.py

```diff
@@ -9,6 +9,7 @@
 DATA_PATH = HERE / "data"
 
 dt_dec_16 = datetime.now(timezone("US/Pacific")).replace(2023, 12, 16, 21, 0, 0)
+dt_april_8 =datetime.now(timezone("US/Pacific")).replace(hour=11, minute=7, month=4, day=8)
```

Comment:
can you specify the `second` here too? maybe even just copy the format of the previous date:
```
replace(2024, 4, 8, 11, 7, 0)
```
Without the seconds specified, this test could be flaky depending on the time the test is run